examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2097,200 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
 
 	atomic.StoreInt32(&stop, 1)
 }
+
+// AggregationExamples contains examples of aggregation operations.
+func AggregationExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	coll := db.Collection("sales")
+	coll1 := db.Collection("air_alliances")
+
+	err := coll.Drop(context.Background())
+	require.NoError(t, err)
+	err1 := coll1.Drop(context.Background())
+	require.NoError(t, err1)
+
+	{
+		// Start Aggregation Example 1
+		pipeline := bson.A{
+			bson.D{{"$match", bson.D{{"items.fruit", "banana"}}}},
+			bson.D{{"$sort", bson.D{{"date", 1}}}},
+		}
+
+		cs, err := coll.Aggregate(ctx, pipeline)
+		require.NoError(t, err)
+		defer cs.Close(ctx)
+
+		ok := cs.Next(ctx)
+		next := cs.Current
+
+		// End Aggregation Example 1
+
+		require.True(t, ok)
+		require.NoError(t, err)
```

Comment:
The error should always be checked before using the result. If `Aggregate` errors, it returns `(nil, error)`, so the uses of `cs` above would panic.

Reply:
`err` is actually checked before using `cs` on line 2121, too. This `require` is definitely redundant, though, as `err` does not get redefined within those nine lines, so I can probably remove it.